data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```diff
@@ -350,6 +376,25 @@
     "flags": [ "SEES", "SMELLS", "HEARS", "PATH_AVOID_DANGER_1" ],
     "reproduction": { "baby_egg": "cookies", "baby_count": 10, "baby_timer": 5 }
   },
+  {
+    "id": "mon_cookie_hydra",
+    "type": "MONSTER",
+    "name": { "str": "cookie hydra" },
+    "description": "A giant cookie dragon with multiple heads.  It prowls looking for food.",
```

Comment:
Maybe it should have a pup version? I might be a bit fast if it goes straight from egg to giant hydra.